test/data/schemas.test.ts

```diff
@@ -206,25 +205,6 @@ test('state incentives JSON files match schemas', async tap => {
         );
       }
 
-      if (incentive.authority_type === AuthorityType.Other) {
-        tap.hasProp(
-          incentive,
-          'eligible_geo_group',
-          `authority_type 'other' must include a geo group (id ${incentive.id})`,
-        );
-
-        tap.hasProp(
-          stateGeoGroups,
-          incentive.eligible_geo_group!,
-          `nonexistent geo_group (${stateId}, id ${incentive.id})`,
-        );
-      } else {
-        tap.notOk(
-          incentive.eligible_geo_group,
-          `only authority_type 'other' can have a geo group (id ${incentive.id})`,
-        );
-      }
-
```

Comment:
Just flagging that I moved these into the new test file to keep geogroup-related tests together. The middle test (nonexistent group ID) is implicitly covered by the test that verifies equivalence of config and incentives.